src/std/rpc/mod.rs

```diff
@@ -32,20 +32,20 @@ pub enum RpcClientError {
 	Extrinsic(String),
 	#[error("mpsc send Error: {0}")]
 	Send(#[from] std::sync::mpsc::SendError<String>),
+	#[error("Expected some error information, but nothing was found: {0}")]
+	NoErrorInformationFound(String),
 }
 
 #[derive(Debug, PartialEq, Eq, Copy, Clone)]
 pub enum XtStatus {
-	// Todo: some variants to not return a hash with `send_extrinsics`: #175.
-	Finalized,
-	InBlock,
-	Broadcast,
-	Ready,
-	Future,
-	/// uses `author_submit`
-	SubmitOnly,
-	Error,
-	Unknown,
+	Future = 10,
+	Finalized = 5,
+	InBlock = 4,
+	Broadcast = 3,
+	Ready = 2,
+	/// uses `author_submit` without watching.
+	SubmitOnly = 1,
+	Unknown = 0,
```

Comment:
I would vote to sort this according to the numbering. :)

Reply:
Done 🍡